SQL/02.CreateTables.sql

```diff
@@ -0,0 +1,250 @@
+		- CHECK constraint must be included in CREATE TABLE instruction after columns definition in format:
+			CONSTRAINT <check_name>_chk CHECK (<Expresion_to_check>).
+
+		- FOREIGN KEY must be added at the last section of the file in format: 
+			ALTER TABLE <table_name>
+			ADD CONSTRAINT <foreign_key_name>_fkey FOREIGN KEY (<column_name>)
+			REFERENCES <foreign_table_name> (<foreign_column_name>) MATCH SIMPLE;
+		
+		- Names consisted of more than 1 word must use '_' as words separator.
+			Object names such as tables, constraints, functions are not case sensitive,
+			so to make them easy easy-readable please use word separator.
+
+		- Constraints' suffixes:
+			- PRIMARY KEY 	<- suffix '_pkey'
+			- FOREIGN KEY 	<- suffix '_fkey'
+			- CHECK 		<- suffix '_chk'
+
+		- Constraints' names:
+			- PRIMARY KEY 	<- must have same name as table name + appropriate suffix.
+			- FOREIGN KEY 	<- must have same name as column, which it is related to, 
+								skipping column name suffix which for fkeys must be '_ID' + appropriate suffix.
+			- CHECK 		<- must have same name as column, which it is related to + appropriate suffix.
+								If CHECK is related to more then 1 column, name can be custom to describe,
+								what does it check + appropriate suffix.
+
+
+    .NOTES
+
+        Version:            1.2
+        Author:             Stanisław Horna
+        Mail:               stanislawhorna@outlook.com
+        GitHub Repository:  https://github.com/PLProjektKompetencyjny/PK_6IO1z_Projekt4_DataBase
+        Creation Date:      16-Mar-2024
+        ChangeLog:
+
+        Date            Who                     What
+		2024-03-17		Stanisław Horna			Removed Description column from "dict_" tables.
+												Unified naming:
+													- word separation in names with '_'
+													- table names changed to singular form
+													- columns with foreign keys ends with '_ID'
+												Basic check constraints and Foreign keys added.
+												CASCADE added to each DROP instruction, to remove all related objects
+
+		2024-03-18		Stanisław Horna			Re-desing of address, customer, admin tables.
+						Grzegorz Kubicki		Switched to approach where login credentials,
+												for both customers and admins are stored in user_account table.
+												User_details store customer related information, 
+												which are not required for admin account
+
+
+*/
+
+-- Drop existing tables
+DROP TABLE IF EXISTS User_account CASCADE;
+DROP TABLE IF EXISTS User_details CASCADE;
+DROP TABLE IF EXISTS Reservation CASCADE;
+DROP TABLE IF EXISTS Reservation_Room CASCADE;
+DROP TABLE IF EXISTS dict_reservation_status CASCADE;
+DROP TABLE IF EXISTS Invoice CASCADE;
+DROP TABLE IF EXISTS dict_invoice_status CASCADE;
+DROP TABLE IF EXISTS Room CASCADE;
+DROP TABLE IF EXISTS Room_Type CASCADE;
+DROP TABLE IF EXISTS dict_room_status CASCADE;
+
+
+-- Create tables
+CREATE TABLE User_account (
+	ID serial primary key NOT NULL,
+	user_name varchar NOT NULL,
```

Comment:
Wiem że postgres i tak sobie to zmienia, ale zrobiłbym duże literki, żeby zachować spójność

Reply:
Ładniej będzie wyglądać, poprawie

Reply:
[e2c6deb](https://github.com/PLProjektKompetencyjny/PK_6IO1z_Projekt4_DataBase/pull/18/commits/e2c6deb05a49993838f4312d031e27d72314f3c0)